Make fails on MacOS (11.0.1)

Running `make` on MacOS (Intel chip) gives an error:

```
tcp-keepalive-test.c:36:36: error: use of undeclared identifier 'TCP_KEEPIDLE'
    if (setsockopt(s, IPPROTO_TCP, TCP_KEEPIDLE,
                                   ^
```

CC Version info: 
```
Apple clang version 12.0.0 (clang-1200.0.32.27)
Target: x86_64-apple-darwin20.1.0
Thread model: posix
```

--- a/tcp-keepalive-test.c
+++ b/tcp-keepalive-test.c
@@ -10,6 +10,9 @@
 #include <arpa/inet.h>
 #include <netinet/tcp.h>
 
+#ifdef __APPLE__
+#define TCP_KEEPIDLE TCP_KEEPALIVE
+#endif
 
 int do_connect(struct sockaddr_in *dst, int keepalive_time) {
     int s = socket(AF_INET, SOCK_STREAM, IPPROTO_TCP);

